operations/tasks/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 - include: upgrade-docker.yml
   when: osbs_upgrade_docker
+
+- include: update-node_selector-labels.yml
```

Comment:
I just noticed this, let's rename the file so it's not a mix of underscore kebab case. Let's use dashes (kebab)

Reply:
ok